core-java-modules/core-java-uuid/src/main/java/com/baeldung/uuid/UUIDGenerator.java

```diff
@@ -12,9 +12,28 @@
 public final class UUIDGenerator {
 
     private static final char[] hexArray = "0123456789ABCDEF".toCharArray();
+    private static final Random random = new Random();
 
     private UUIDGenerator() {}
 
+    // ref: https://datatracker.ietf.org/doc/html/rfc4122#appendix-C
+    public enum Namespace {
+        DNS("6ba7b810-9dad-11d1-80b4-00c04fd430c8"),
+        URL("6ba7b811-9dad-11d1-80b4-00c04fd430c8"),
+        OID("6ba7b812-9dad-11d1-80b4-00c04fd430c8"),
+        X500("6ba7b814-9dad-11d1-80b4-00c04fd430c8");
+
+        public final String value;
```

Comment:
I use `public final` because this `value` can be used in `generateType5UUID(String namespace, String name)` method in this class.